cardano-node/src/Cardano/Node/Tracing/Tracers.hs

```diff
@@ -363,6 +363,13 @@ mkNodeToClientTracers trBase trForward mbTrEKG _trDataPoint trConfig = do
                 severityTChainSync
                 allPublic
     configureTracers trConfig docTChainSync [chainSyncTr]
+    txMonitorTr  <-  mkCardanoTracer
+                trBase trForward mbTrEKG
```

Comment:
Can you fix the indent here? It should be two spaces:
```
mkCardanoTracer
 trBase trForward mbTrEKG
 "TxMonitorClient"
  ...
```

Reply:
This is consistent with the surrounding.  I'll change it consistently in the code around.

Reply:
I pushed a commit which fixes it.